src/wrappers/themis/wasm/package.json

```diff
@@ -37,12 +37,12 @@
   },
   "homepage": "https://www.cossacklabs.com/themis/",
   "devDependencies": {
-    "@types/emscripten": "^1.39.4",
+    "@types/emscripten": "^1.39.6",
     "@types/expect": "^24.3.0",
-    "@types/mocha": "^8.2.2",
-    "@types/node": "^14.14.36",
-    "mocha": "^7.1.1",
-    "ts-mocha": "^8.0.0",
-    "typescript": "^4.2.3"
+    "@types/mocha": "^9.1.0",
+    "@types/node": "^17.0.23",
+    "mocha": "^9.2.2",
+    "ts-mocha": "^9.0.2",
+    "typescript": "^4.6.3"
```

Comment:
Seems Node.js v8 does not like the `mocha` upgrade.

How about reverting this, then doing
```
npm audit --fix
```
from scratch?